apps/charterafrica/src/payload/globals/Navigation.js

```diff
@@ -38,6 +43,40 @@ const Navigation = {
     read: () => true,
   },
   fields: [
+    {
+      name: "logo",
+      label: {
+        en: "Logo",
+        pt: "Imagem",
+        fr: "Logo",
+      },
+      type: "group",
+      fields: [
+        {
+          name: "image",
+          type: "upload",
+          relationTo: "media",
+          required: true,
+          filterOptions: {
+            mimeType: { contains: "image" },
+          },
+          label: {
+            en: "Image",
+            pt: "Imagem",
+            fr: "Logo",
+          },
+        },
+        link({
```

Comment:
Not sure why we need to link this to any other page other than `/`

Reply:
👇🏽 is how the NavBar looks in prod

<img width="1193" alt="Screenshot 2023-07-17 at 12 55 37" src="https://github.com/CodeForAfrica/ui/assets/1779590/92048be5-0101-4bd1-bfa6-965c6802a838">

Doesn't have to but `logo` and `HOME` could point to different links.